### PR TITLE
Updated index file

### PR DESCRIPTION
**Pull request by jm-trd-ms**
Updated content in index.yml.  Updated links.

### CODE REVIEW

File: microsoft-edge/index.yml [hunk 1]
```diff
@@ -20,9 +20,13 @@ highlightedContent:
 #           whats-new
   items:
     # Card
-    - title: "Download and Install: Microsoft Edge Channels"
+    - title: "Download Microsoft Edge (Official Stable Channel)"
```
Comment (Reezaali) [inline microsoft-edge/index.yml]: Do we need the word Official?

File: microsoft-edge/index.yml [hunk 2]
```diff
@@ -20,9 +20,13 @@ highlightedContent:
...
       itemType: download
-      url: https://www.microsoftedgeinsider.com/download
+      url: https://www.microsoft.com/edge
+    # Card
+    - title: "Download Microsoft Edge Insider Channels"
+      itemType: download
```
Comment (Reezaali) [inline microsoft-edge/index.yml]: Is download the right itemType?

File: microsoft-edge/index.yml (hunk 1 above)
Comment (jm-trd-ms) [reply microsoft-edge/index.yml]: The actual branch is called Official.  
image

**Comment by jm-trd-ms** [comment]
#sign-off and merge.

**Pull request merged**